### PR TITLE
#43 - create choose goals page

### PR DESCRIPTION
**Pull request by krichdev**
Pulled in motivation flow into index page as well as goals

### CODE REVIEW

File: shared/generated/binding.ts [hunk 1]
```diff
@@ -4,12 +4,9 @@ import { IResolvers } from 'graphql-tools/dist/Interfaces'
 import * as schema from  './schema'
 
 export interface Query {
-    activity: <T = Array<activity>>(args: { distinct_on?: Array<activity_select_column> | null, limit?: Int | null, offset?: Int | null, order_by?: Array<activity_order_by> | null, where?: activity_bool_exp | null }, info?: GraphQLResolveInfo | string, options?: Options) => Promise<T> ,
```
Comment (andycmaj) [inline shared/generated/binding.ts]: these `generated` changes should be reverted... looks like you ran generate when your local postgres db was not up to date so it removed a few types.

File: ui/components/UserMotivationsCard.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  background: grey;
+  padding: 10px;
+  display: flex;
+  justify-content: center;
+  margin-top: 0;
+  color: white;
+`;
+
+const UserMotivationsCard = motivation => {
+  const motivationGoals = motivation.motivation.motivationGoals;
```
Comment (andycmaj) [inline ui/components/UserMotivationsCard.tsx]: im surprised you need to do `motivation.motivation` since you're already dereferencing this when you add the component below `<UserMotivationsCard motivation={userMotivation.motivation} />`.

wouldn't this end up being `userMotivation.motivation.motivation.motivationGoals`? is that really what the gql response looks like?

File: ui/components/UserMotivationsCard.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <MotivationCard>
+      <GoalHeader>{motivation.motivation.id}</GoalHeader>
+      {motivation.motivation.motivationGoals.map(motivationGoal =>
```
Comment (andycmaj) [inline ui/components/UserMotivationsCard.tsx]: you already have `const motivationGoals` above.

File: shared/generated/binding.ts (hunk 1 above)
Comment (krichdev) [reply shared/generated/binding.ts]: reverted binding.ts and schema.graphql - i believe it's ok for the index in generated to be diff since I added a mutation?

File: ui/graphql/index.tsx [hunk 4]
```diff
@@ -10,357 +10,10 @@ export type Scalars = {
   Boolean: boolean,
   Int: number,
   Float: number,
-  jsonb: any,
   timestamptz: any,
   json: any,
 };
 
-/** columns and relationships of "activity" */
-export type Activity = {
```
Comment (andycmaj) [inline ui/graphql/index.tsx]: also need to revert this change
